src/main/java/lotto/domain/stats/LottoEarningRateCalculator.java

```diff
@@ -28,7 +28,7 @@ private double eval() {
         long prizeTotal = 0L;
 
         for (Prize prize : Prize.values()) {
-            Long winnings = lottoScoreBoard.getWinningsByEqualNumberCount(prize.getEqualNumberCount());
+            Long winnings = lottoScoreBoard.getWinnerCountByPrize(prize);
```

Comment:
```suggestion
            long winnings = lottoScoreBoard.getWinnerCountByPrize(prize);
```
불필요한 Boxing 이 일어나네요 😀